backend/python/app/services/implementations/story_service.py

```diff
@@ -267,6 +268,30 @@ def update_story_translation_contents(self, story_translation_contents):
             )
             raise error
 
+    def update_story_translation_stage(self, story_translation_data):
+        story_translation = StoryTranslation.query.filter_by(
+            id=story_translation_data["id"]
+        ).first()
+        new_stage = story_translation_data["stage"]
+        user_id = get_user_id_from_request()
```

Comment:
urhg. I couldnt get this mutation working while testing from graphql interface Altair. The issue is that `get_user_id_from_request` returns a string because of `auth_service.user_service.get_user_id_by_auth_id` returning a string. We cannot simply just modify `auth_service.user_service.get_user_id_by_auth_id` to return an int as it's used in multiple places and might cause unforseen issues. 
Instead, for now, use `user_id = int(get_user_id_from_request())` and leave a todo to update the line when `get_user_id_from_request` is updated. I'll work on updating `get_user_id_from_request`